app/__init__.py

```diff
@@ -1,51 +1,29 @@
-from flasgger import Swagger
 from flask import Flask
-from flask_cors import CORS
-from flask_login import LoginManager
-from flask_migrate import Migrate
-from flask_restplus import Api
-from flask_sqlalchemy import SQLAlchemy
-from sqlalchemy import MetaData
 
 from config import Config
 
-application = Flask(__name__)
-CORS(application, supports_credentials=True)
-
-swagger = Swagger(application)
-
-application.config.from_object(Config)
-
-naming_convention = {
-    'ix': 'ix_%(column_0_label)s',
-    'uq': 'uq_%(table_name)s_%(column_0_name)s',
-    'ck': 'ck_%(table_name)s_%(column_0_name)s',
-    'fk': 'fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s',
-    'pk': 'pk_%(table_name)s',
-}
-
-# db = SQLAlchemy(app=application)
-db = SQLAlchemy(app=application, metadata=MetaData(naming_convention=naming_convention))
-
-migrate = Migrate(application, db)
-with application.app_context():
-    if db.engine.url.drivername == 'sqlite':
-        migrate.init_app(application, db, render_as_batch=True)
-    else:
-        migrate.init_app(application, db)
-
-login = LoginManager(application)
-
-api = Api(
-    application,
-    version='1.0',
-    title='HuXin API',
-    description='Huxin API v1',
-    doc='/doc/',
-)
-
-from app import models  # noqa
-from app import auth_api  # noqa
-from app import complaint_api  # noqa
-from app import merchant_query_api  # noqa
-from app import comment_api  # noqa
+
+def create_app(**kwargs):
+    """
+    Entry point to the Flask RESTful Server application
+    ===================================================
+    """
+    # This is a workaround for Alpine Linux (musl libc) quirk:
+    # https://github.com/docker-library/python/issues/211
+    import threading
+
+    threading.stack_size(2 * 1024 * 1024)
```

Comment:
This change is beneficial to docker only? are you using docker as development environment? @e-cloud

Reply:
And why 2M, this number?

Reply:
no, just some copy pasted code from reference library. I could disable it.